walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +276,181 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
+  
+  // Newly added tests below
+ 
+  @Test
+  public void everyAttendeeIsConsideredWithOptionalFullDay() {
+    // Have each person have different events. 
+    // Optional guest C has an all day event. 
+    // We should see two options because each person has
+    // split the restricted times. Guest C is not invited. 
+    //
+    // Optional: |-------------C---------------|
+    // Events  :       |--A--|     |--B--|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+ 
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.fromStartDuration(TIME_0000AM, DURATION_24_HOUR),
+            Arrays.asList(PERSON_C)));
+ 
+    MeetingRequest request =
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+      
+    request.addOptionalAttendee(PERSON_C); 
+ 
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
```

Comment:
We can have a line break before TimeRange.

ex:
Arrays.asList(
    TimeRange....,
    TimeRange...,
    TimeRange...);

Reply:
(For some reason GitHub does not show the spaces)